pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -687,6 +687,22 @@ g {
 }
 nodes_that_need_write = []
 
+g {
+	n[0]:  alloc      _     => _2  @ bb1[2]:  fn test_ref_field; _2 = pointers::calloc(move _3, move _4)  
+	n[1]:  copy       n[0]  => _1  @ bb2[3]:  fn test_ref_field; _1 = move _2 as *mut pointers::S (Misc)  
```

Comment:
What does `(Misc)` mean?

Reply:
And in general the other things in the parentheses after the type.

Reply:
That's the [`CastKind`](https://doc.rust-lang.org/stable/nightly-rustc/rustc_middle/mir/enum.CastKind.html).  There are a few kinds of pointer coercions that have specific `CastKind`s, and everything else is `CastKind::Misc`.